db/db_test.cc

```diff
@@ -4099,6 +4099,42 @@ TEST_F(DBTest, ConcurrentFlushWAL) {
   }
 }
 
+TEST_F(DBTest, ManualFlushWalAndWriteRace) {
```

Comment:
iiuc, this test failure will be caught with a probability. Maybe worth a comment here.

Reply:
Thanks for review. Already add a comments.